seatunnel-examples/seatunnel-spark-examples/src/main/java/org/apache/seatunnel/test/spark/LocalTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.test.spark;
```

Comment:
hi @felix-thinkingdata, 

- the package name `package org.apache.seatunnel.example.spark;` is better
- the class name `LocalSparkExample` is better

Can you help to rename the `flink` example at this patch?

Reply:
> 

ok ，Let me modify that. Should I revise flink's example together

Reply:
> ok ，Let me modify that. Should I revise flink's example together

yeah.

Reply:
> > ok ，Let me modify that. Should I revise flink's example together
> 
> yeah.

done.